.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           - "++2.12.10 testsJVM/test"
           - "++2.12.11 testsJVM/test"
           - "++2.12.11 testsJS/test"
+          - "++2.12.12 testsJS/test"
           - "++2.13.1 testsJVM/test"
           - "++2.13.2 testsJVM/test"
           - "++2.13.3 testsJVM/test"
```

Comment:
a few lines below is:
`run: sbt ++2.12.11 testsJVM/test` 
this should also probably be updated to `2.12.12`?